main.js

```diff
@@ -128,25 +128,25 @@ app.get('/pay_ticket/:id/:rate', function (req, resp) {
                 var exitTime = enterTime.addHours(req.params.rate);
                 connection.query('UPDATE tickets SET exit_time=? WHERE id=?',[exitTime, req.params.id],
                     function (error2) {
-                   if(error2){
-                       console.log(error2);
-                   }
+                        if(error2){
+                            console.log(error2);
+                        }
 
-                });
+                    });
 
                 // Update the payment in DB
                 connection.query('SELECT * FROM pay_rates WHERE num_of_hours=?', [req.params.rate],
                     function (error3, result) {
-                    if(error3){
-                        console.log(error3);
-                    }
+                        if(error3){
+                            console.log(error3);
```

Comment:
`return` Here to stop the code execution in this block, unexpected behaviour. Also this error is lost to the user and will have unexpected behaviour.